tests/bud.bats

```diff
@@ -3171,6 +3171,10 @@ _EOF
   run_buildah build --add-host=myhostname:$ip -t testbud \
                   --signature-policy ${TESTSDIR}/policy.json --file ${mytmpdir} .
   expect_output --from="${lines[2]}" --substring "^$ip\s+myhostname"
+
+  run_buildah 1 build --no-cache --add-host=myhostname:$ip \
+                  --no-hosts \
```

Comment:
What error does/should this produce?  Can we check that it's the error message we're expecting?  Should we be complaining that `--add-host` and `--no-hosts` conflict with each other?

Reply:
Fixed.